.classpath

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Unfortunately, as files were moved around, it seems that deltas were only captured meaningfully in a few cases. 

The schema was not changed much. Very minor things. Like hoisting the encoding property to the base format to remove it from where it was repeatedly.